### PR TITLE
Improve Duster glob support using Symfony finder

### PR DESCRIPTION
**Pull request by driftingly**
This PR updates the Duster's wildcard support for include/exclude configurations.

Previously a configuration using a wildcard glob would not work:

```json
{
    "exclude": [
        "tests/**/Fixture"
    ]
}
```

This PR runs these patterns through Symfony's finder and falls back to the previous `glob` method.

### CODE REVIEW

File: app/Support/DusterConfig.php [hunk 1]
```diff
@@ -63,11 +66,39 @@ public static function scopeConfigPaths(array $config): array
...
+        return collect($paths)->flatMap(fn ($path) => static::globPath($path))->values()->toArray();
+    }
+
+    /**
+     * @return  array<int, string>
+     */
+    public static function globPath(string $path): array
+    {
+        try {
+            if (Str::of($path)->endsWith('.php')) {
```
Comment (driftingly) [inline app/Support/DusterConfig.php:78]: May require other checks, but this is a start

**Comment by mattstauffer** [comment]
@driftingly This bumps us from Tlint 8 to Tlint 9, which changes PHP requirement from 8.0 to 8.1, so this'll need to be a minor or even possibly major release, can't remember which we're allowed to increase dependencies in.

**Comment by driftingly** [comment]
@mattstauffer Duster already requires PHP 8.1 since Pint 1.6+ requires 8.1 and was the reason we went from 1.x to 2.x. 
I did release this as a minor because there were a lot of other dependency updates.

**Comment by benholmen** [comment]
FYI this PR caused a regression for me. My duster.json file was:

```json
{
    "include": [],
    "exclude": ["bootstrap/*", "app/Http/Controllers/Auth/*"]
}
```

but after upgrading v2.6.0 => v2.7.1, I started getting lint errors on `bootstrap.php` and an Auth controller.

I updated my config to the following and it worked:

```json
{
    "include": [],
    "exclude": ["bootstrap", "app/Http/Controllers/Auth"]
}
```

**Pull request merged**